AsyncDisplayKit/Private/ASStackUnpositionedLayout.mm

```diff
@@ -363,17 +363,23 @@ static void flexChildrenAlongStackDimension(std::vector<ASStackUnpositionedItem>
     }
     return;
   }
-  std::function<CGFloat(const ASStackUnpositionedItem &, BOOL)> flexAdjustment = flexAdjustmentInViolationDirection(items,
-                                                                                                                    style,
-                                                                                                                    violation,
-                                                                                                                    flexFactorSum);
+  std::function<CGFloat(const ASStackUnpositionedItem &)> flexAdjustment = flexAdjustmentInViolationDirection(items,
+                                                                                                              style,
+                                                                                                              violation,
+                                                                                                              flexFactorSum);
+
+  // Compute any remaining violation. Any remaining violation it will be allocated to the first flexible child.
+  const CGFloat remainingViolation = std::accumulate(items.begin(), items.end(), violation, [&](CGFloat x, const ASStackUnpositionedItem &item) {
+    return x - flexAdjustment(item);
+  });
   BOOL isFirstFlex = YES;
```

Comment:
is `isFirstFlex` always YES or am I missing something? If so can we just factor it out or at least make it a const?